demo/admin/package.json

```diff
@@ -77,15 +54,38 @@
         "@types/react-router-dom": "^5.0.0",
         "@vitejs/plugin-react-swc": "^3.7.2",
         "chokidar-cli": "^3.0.0",
+        "cosmiconfig-toml-loader": "^1.0.0",
```

Comment:
Why is this needed?

Reply:
demo/admin needs this dependency for @graphql-codegen/cli
<img width="376" alt="Screenshot 2025-01-29 at 09 01 15" src="https://github.com/user-attachments/assets/37a21ba4-709a-4f8f-af34-014a197a614d" />